subprojects/distributions/src/integTest/groovy/org/gradle/DistributionIntegritySpec.groovy

```diff
@@ -191,6 +193,32 @@ class DistributionIntegritySpec extends DistributionIntegrationSpec {
         assert errors.empty
     }
 
+    @Requires(TestPrecondition.LINUX)
+    def "validate dependency archives"() {
```

Comment:
Let's mention the issue this originated from here.

Reply:
Done